config/google_cloud_exporter/bigip_f5/README.md

```diff
@@ -0,0 +1,48 @@
+# Big IP F5 with Google Cloud
+
+The bigip Receiver can be used to metrics to Google Cloud Monitoring.
+
+## Prerequisites
+
+See the [prerequisites](../README.md) doc for Google Cloud prerequisites.
+
+## Limitations
+
+To avoid duplicate metrics, ensure each collector is scraping a single f5 system. This ensures
```

Comment:
minor, should we use F5 over f5? also, should we be calling it a BIG-IP system instead of F5 system?

Reply:
Yes. I have a hard time calling it "big ip" after using "f5" all these years. I renamed everything to bigip.